script/update-content-versions-and-check-for-archived-slugs.ts

```diff
@@ -91,11 +92,16 @@ async function processBook() {
       throw error;
     });
 
-  books[args.bookId] = { defaultVersion: version };
+  const updatedBooksConfig = { ...books };
+  updatedBooksConfig[args.bookId] = { defaultVersion: version };
```

Comment:
Shouldn't the `updatedBooksConfig` be in `fs.writeFileSync`?

Reply:
ofc it should, thank :facepalm: